oximeter/oximeter/schema/switch-rib.toml

```diff
@@ -0,0 +1,30 @@
+format_version = 1
+
+[target]
+name = "switch_rib"
+description = "Maghemite router routing information base"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "hostname", "link_id", "rack_id" ] }
```

Comment:
is `link_id` supposed to be `sled_id` here?

Reply:
Yeah, my bad, thank you for catching. Fixed in 2823506b00d48c661911faca9781c50e5ed35377